cypress/integration/menu/menu.js

```diff
@@ -1,21 +1,39 @@
-import { Given, When, Then, Before } from "cypress-cucumber-preprocessor/steps"
+import { Given, When, Then, Before } from 'cypress-cucumber-preprocessor/steps';
 import { v4 as uuidv4 } from 'uuid';
 
 const USER_NAME = 'testUser@unit.no';
 
-Before(async () => {
-  cy.addUser(USER_NAME).as('authResult');
+Before(() => {
+  cy.addUser(USER_NAME).then((authResult) => {
+    cy.wrap(authResult).as('authResult');
+  });
 });
 
 Given('that the user is not logged in', () => {
-  cy.visit('/');
-  cy.get('[data-testid=menu-login-button]').should('be.visible');
+  cy.get('@authResult').then((authResult) => {
+    const auth = `Bearer ${authResult.AccessToken}`;

```

Comment:
hvorfor må du ha authorization header her?

Reply:
vil en bruker som ikke er logget inn ha accessToken? 🤔

Reply:
Burde ha brukt en feature hvor man må være pålogget, dette er for å vise hvordan det skal brukes.